README.md

```diff
@@ -1,2 +1,60 @@
-# BookApp
-Project: Library
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Library JS Project
+
+> This project features a list of items that can be added through a form, which can be hidden by a button.
+A 'Read' button will show Red if the book is marked 'not read', and green if marked 'read'.
+
+![screenshot](library.png)
+
+## Built With
+
+HTML5
+JavaScript
+
+### Repo
+
+The repo is available in [this link](https://github.com/RokoVarano/BookApp).
```

Comment:
- This link does not open the app. Fix that.

Reply:
Link has been changed to live Git Hub Pages